src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -1052,11 +1052,13 @@ public function foreignIdFor($model, $column = null)
      * Create a new float column on the table.
      *
      * @param  string  $column
-     * @param  int  $precision
+     * @param  int|null  $precision
      * @return \Illuminate\Database\Schema\ColumnDefinition
      */
     public function float($column, $precision = 53)
```

Comment:
Why not default `$precision` to `null`?

As the newly added `Builder::$defaultFloatPrecision` equals the previous value by default, changing the default value of the `$precision` argument wouldn't be a breaking change, even for any classes overriding this method.